### ISSUE TITLE
sys.path. not providing completion for `append`

### ISSUE THREAD

**Issue by DonJayamanne** [Microsoft/python-language-server#728]
@kaisbe commented on [Fri Mar 08 2019](https://github.com/Microsoft/vscode-python/issues/4682)

Hello,

I'm requesting a new feature / enhancement, which is about providing suggestions of available directories, in order to make it easy to set the interpreter search path through auto-completion.

Best Regards.

## Environment data

- VS Code version: 1.30.2
- Extension version: 2019.1.0
- Language Server: Microsoft Python Language Server (2019.1.1)
- OS and version: Linux 4.15.0-33-generic
- Python version: Python 2.7.15+
- Type of virtual environment used: N/A

## Expected behaviour

Using the TypeScript extensions when I try to import a local module:
1. I begin the search path with './'
2. I press Ctrl+Space
3. the extension lists the available modules + directories

As a Python programmer, I need also to get the python extension to make my life easy, through providing the available directories' suggestions for auto-completing the interpreter search path I'm trying to specify (similar behavior as the TypeScript extension's one).

So to sum up things briefly, this is the behavior I'm expecting:
1. I type `sys.path.append(`
2. I begin typing the search path
3. the extension provides me suggestions as a list of the possible/available search paths

## Actual behaviour

1. I type `sys.path.`
2. I press Ctrl+Space
3. **The method `append()` doesn't get suggested**
4. I type manually `append(`
5. I begin typing the search path
6. I wait
7. I even press on Ctrl+Space
8. No suggestions appear, and only irrelevant keywords get listed

**Comment by DonJayamanne** [comment]
We discussed this in @Microsoft/pyvsc and are suggesting that we not provide auto completion for paths in `sys.path`.

**Comment by MikhailArkhipov** [comment]
LS 0.2.x

![image](https://user-images.githubusercontent.com/12820357/54237503-5c5e8f80-44d3-11e9-8578-55d4abadaa4d.png)

You can try in in the beta channel.
"python.analysis.downloadChannel": "beta"
